src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/LanguageServices/VsContainedLanguageComponentsFactoryBase.cs

```diff
@@ -49,6 +67,8 @@ internal abstract class VsContainedLanguageComponentsFactoryBase : IVsContainedL
             {
                 await ProjectServices.ThreadingService.JoinableTaskFactory.SwitchToMainThreadAsync();
 
+                await InitializeAsync().ConfigureAwait(true);
```

Comment:
@333fred I have reverted my last commit - LS Host initialization needs to be done on the UI thread, otherwise the test is throwing an exception.